nexus/tests/integration_tests/endpoints.rs

```diff
@@ -1280,15 +1283,6 @@ lazy_static! {
             ],
         },
 
-        VerifyEndpoint {
-            url: "/by-id/instances/{id}",
-            visibility: Visibility::Protected,
-            unprivileged_access: UnprivilegedAccess::None,
-            allowed_methods: vec![
-                AllowedMethod::Get,
-            ],
-        },
-
```

Comment:
Given that the endpoints are now collapsed this mechanism of having two `VerifyEndpoint` definitions won't work. Essentially any endpoint that is verified here will be consumed from the spec so if you try to verify the same endpoint twice you'll end up with a situation where it says the spec doesn't contain the other instance of the endpoint (because the previous verify consumed it). 

After the API is converted this suite may need to be revisited.